### PR TITLE
docs(tutorials):  review + fixes + faq EM - MTA-1688c

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: tutorials/deploy-instances-packer-terraform/index.mdx [hunk 1]
```diff
@@ -24,7 +24,7 @@ Both applications are available for Linux, MacOS, Windows, FreeBSD and NetBSD.
 
 ## Downloading and Installing Packer
 
-1. Download [Packer](https://www.packer.io/downloads.html) for your Operating system. If you want to have some specific configuration, you can also [build the application](https://www.packer.io/intro/getting-started/install.html#compiling-from-source) from source.
+1. Download [Packer](https://www.packer.io/downloads.html) for your Operating System. If you want to define specific configuration, you can also [build the application](https://www.packer.io/intro/getting-started/install.html#compiling-from-source) from source.
```
Comment (RoRoJ) [inline tutorials/deploy-instances-packer-terraform/index.mdx:27]:
```suggestion
1. Download [Packer](https://www.packer.io/downloads.html) for your Operating System. If you want to define a specific configuration, you can also [build the application](https://www.packer.io/intro/getting-started/install.html#compiling-from-source) from source.
```

File: tutorials/prometheus-monitoring-grafana-dashboard/index.mdx [hunk 2]
```diff
@@ -24,29 +24,28 @@ This tutorial explains how to use a Prometheus Monitoring server with a Grafana
...
+- You have a [Scaleway Instance](/compute/instances/how-to/create-an-instance/) running on [Ubuntu Focal Fossa (20.04)](https://www.ubuntu.com/)
 
 </Message>
 
 ## Preparing your environment
 
-In this tutorial, we use an Instance running on [Ubuntu Focal Fossa (20.04))](https://www.ubuntu.com/).
-
-1. To run Prometheus safely on our server, we have to create a user for Prometheus and Node Exporter without the possibility to log in.
+1. We have to create a user for Prometheus and Node Exporter without the possibility to log in.
```
Comment (RoRoJ) [inline tutorials/prometheus-monitoring-grafana-dashboard/index.mdx:33]: Could this explanatory bit all go in the text before step one, and then step 1 be "Run the following command to create the user":

File: tutorials/deploy-instances-packer-terraform/index.mdx (hunk 1 above)
Comment (ofranc) [inline tutorials/deploy-instances-packer-terraform/index.mdx:27]: on avait dit qu'on mettait des majuscules à Operating System ?

File: tutorials/prometheus-monitoring-grafana-dashboard/index.mdx [hunk 3]
```diff
@@ -103,27 +102,31 @@ As your Prometheus is only capable of collecting metrics, we want to extend its
   WantedBy=multi-user.target
   ```
 
-7. Collectors are used to gather information about the system. By default a set of collectors is activated. You can see the details about the set in the [README-file](https://github.com/prometheus/node_exporter/blob/master/README.md#enabled-by-default). If you want to use a specific set of collectors, you can define them in the `ExecStart` section of the service. Collectors are enabled by providing a` --collector.<name>` flag. Collectors that are enabled by default can be disabled by providing a `--no-collector.<name>` flag.
+  <Message type="tip">
+
+  Collectors are used to gather information about the system. By default a set of collectors is activated. You can see the details about the set in the [README-file](https://github.com/prometheus/node_exporter/blob/master/README.md#enabled-by-default). If you want to use a specific set of collectors, you can define them in the `ExecStart` section of the service. Collectors are enabled by providing a` --collector.<name>` flag. Collectors that are enabled by default can be disabled by providing a `--no-collector.<name>` flag.
```
Comment (ofranc) [inline tutorials/prometheus-monitoring-grafana-dashboard/index.mdx:107]:
```suggestion
  Collectors are used to gather information about the system. By default, a set of collectors is activated. You can see the details about the set in the [README file](https://github.com/prometheus/node_exporter/blob/master/README.md#enabled-by-default). If you want to use a specific set of collectors, you can define them in the `ExecStart` section of the service. Collectors are enabled by providing a` --collector.<name>` flag. Collectors that are enabled by default can be disabled by providing a `--no-collector.<name>` flag.
```

**Review by ofranc — APPROVED** [review]

File: tutorials/deploy-instances-packer-terraform/index.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [reply tutorials/deploy-instances-packer-terraform/index.mdx:27]: oui, vu que ç'est un acronym

File: tutorials/prometheus-monitoring-grafana-dashboard/index.mdx [hunk 4]
```diff
@@ -360,26 +363,26 @@ We have now installed Prometheus to monitor your instance.
...
+You can perform queries with the collected data.
 
   <Lightbox src="scaleway-prometheus_graph.png" alt="" />
 
-The interface is very lightweight, and the Prometheus team recommend to use a tool like [Grafana](https://grafana.com/) if you want to do more than testing and debugging the installation.
+The interface we set up is lightweight. The Prometheus team, therefore, recommends using a tool like [Grafana](https://grafana.com/) if you want to do more than testing and debugging the installation.
 
 ## Installing Grafana
 
 1. Install Grafana on our instance which queries our Prometheus server.
```
Comment (nedaehm) [inline tutorials/prometheus-monitoring-grafana-dashboard/index.mdx:380]:
```suggestion
1. Install Grafana on our Instance which queries our Prometheus server.
```

**Review by nedaehm — APPROVED** [review]

**Pull request merged**